src/key_gen/message.rs

```diff
@@ -11,7 +11,7 @@ use super::encryptor::{Iv, Key};
 use super::{Acknowledgment, Part};
 use crate::id::PublicId;
 use serde_derive::{Deserialize, Serialize};
-use std::collections::{BTreeMap, BTreeSet};
+use std::collections::{btree_map::BTreeMap, btree_set::BTreeSet};
```

Comment:
`BTreeMap` and `BTreeSet` are exposed directly from `collections`
ie. you can do `use std::collections::{BTreeMap, BTreeSet};`

Reply:
Ah yes, that was my IDE putting in all this. Thanks! :)

Reply:
Ah, @Yoga07 , the diff of the PR shows the `btree_map` & `btree_set` are still added.
can you double check you IDE setting to not put those in? thx